docs/source/user/cli.rst

```diff
@@ -82,61 +82,192 @@ Example output::
+    pexpect                   4.8.0              pyhd3eb1b0_3    defaults
+    pickleshare               0.7.5           pyhd3eb1b0_1003    defaults
+    pip                       22.2.2          py310hecd8cb5_0    defaults
+    prompt-toolkit            3.0.20             pyhd3eb1b0_0    defaults
+    psutil                    5.9.0           py310hca72f7f_0    defaults
+    ptyprocess                0.7.0              pyhd3eb1b0_2    defaults
+    pure_eval                 0.2.2              pyhd3eb1b0_0    defaults
+    pycparser                 2.21               pyhd3eb1b0_0    defaults
+    pygments                  2.11.2             pyhd3eb1b0_0    defaults
+    pyparsing                 3.0.9           py310hecd8cb5_0    defaults
+    python                    3.10.8               h218abb5_1    defaults
+    python-dateutil           2.8.2              pyhd3eb1b0_0    defaults
+    pyyaml                    6.0                      pypi_0    pypi
+    pyzmq                     23.2.0          py310he9d5cce_0    defaults
+    readline                  8.2                  hca72f7f_0    defaults
+    scipy                     1.9.3           py310h09290a1_0    defaults
+    setuptools                65.5.0          py310hecd8cb5_0    defaults
+    six                       1.16.0             pyhd3eb1b0_1    defaults
+    smmap                     4.0.0              pyhd3eb1b0_0    defaults
+    sqlite                    3.40.0               h880c91c_0    defaults
+    stack_data                0.2.0              pyhd3eb1b0_0    defaults
+    tapi                      1000.10.8            ha1b3eb9_0    defaults
+    tbb                       2021.5.0             haf03e11_0    defaults
+    tbb-devel                 2021.5.0             haf03e11_0    defaults
+    tk                        8.6.12               h5d9f67b_0    defaults
+    tornado                   6.2             py310hca72f7f_0    defaults
+    traitlets                 5.1.1              pyhd3eb1b0_0    defaults
+    typing-extensions         4.3.0           py310hecd8cb5_0    defaults
+    typing_extensions         4.3.0           py310hecd8cb5_0    defaults
+    tzdata                    2022f                h04d1e81_0    defaults
+    wcwidth                   0.2.5              pyhd3eb1b0_0    defaults
+    wheel                     0.37.1             pyhd3eb1b0_0    defaults
+    xz                        5.2.6                hca72f7f_0    defaults
+    yaml                      0.2.5                haf1e3a3_0    defaults
+    zeromq                    4.3.4                h23ab428_0    defaults
+    zipp                      3.8.0           py310hecd8cb5_0    defaults
+    zlib                      1.2.13               h4dc903c_0    defaults
+
+    No errors reported.
+
+
+    __Warning log__
+    Warning (cuda): CUDA driver library cannot be found or no CUDA enabled devices are present.
+    Exception class: <class 'numba.cuda.cudadrv.error.CudaSupportError'>
```

Comment:
```suggestion
    (output truncated due to length)
```
Suggest putting this in similarly to how it was originally, not sure that adding all the packages is useful for a sample output.